### PR TITLE
Settings improvements

### PR DESCRIPTION
**Pull request by tsa96**
Been several things I've been wanting to do for ages, plus lots of localization related fixes.

Closes https://github.com/momentum-mod/game/issues/1942
Closes https://github.com/momentum-mod/game/issues/1613
Closes https://github.com/momentum-mod/game/issues/1878
Closes https://github.com/momentum-mod/game/issues/1909
Closes https://github.com/momentum-mod/game/issues/1908 once my red branch is in as well

### CODE REVIEW

File: layout/settings/settings_gameplay.xml [hunk 1]
```diff
@@ -195,47 +212,59 @@
 
 				<!-- TODO: infomessages here in the future. Not doing now because of future safeguard changes -->
 
-				<ChaosSettingsEnumDropDown text="#Settings_Safeguards_PracticeMode" convar="mom_run_safeguard_practicemode">
+				<ChaosSettingsEnumDropDown
+					text="#Settings_Safeguards_PracticeMode"
+					convar="mom_run_safeguard_practicemode"
+					tags="practice"
```
Comment (PeenScreeker) [inline layout/settings/settings_gameplay.xml:218]: Should these safeguards settings each have a tag for safeguards? Or is it sufficient to have the header use that tag only?

File: layout/settings/settings_gameplay.xml [hunk 2]
```diff
@@ -903,8 +1011,9 @@
 
 					<ChaosSettingsEnum
 						text="#Settings_Defrag_HUDFOVEnable"
-						convar="Settings_Defrag_HUDFOVEnable_Info"
+						convar="mom_df_hud_fov_override_enable"
```
Comment (PeenScreeker) [inline layout/settings/settings_gameplay.xml:1014]: Yikes. Good catch!

File: layout/settings/settings_interface.xml [hunk 3]
```diff
@@ -675,7 +689,7 @@
 
 				<Panel class="settings-group__header">
 
-					<Label class="settings-group__title" text="#Settings_Synchronizer_Title" />
+					<Label class="settings-group__title" text="#Settings_Synchronizer_Title" tags="sync,strafe" />
```
Comment (PeenScreeker) [inline layout/settings/settings_interface.xml:692]: I think this should also have a "trainer" tag.

**Review by PeenScreeker — APPROVED** [review]
Very nice QoL improvement

File: layout/settings/settings_gameplay.xml (hunk 1 above)
Comment (tsa96) [reply layout/settings/settings_gameplay.xml:218]: Yep, that's exactly the idea! The header tags are parsed out of the group title labels, that then applies to everything within the group

File: layout/settings/settings_interface.xml (hunk 3 above)
Comment (tsa96) [reply layout/settings/settings_interface.xml:692]: Good call, though I'm on my laptop and about to go to bed. If you wanna add this to this branch this resolve this, be my guest, would be good to get in today

Comment (tsa96) [reply layout/settings/settings_interface.xml]: Added that TAG

**Review by Gocnak — APPROVED** [review]

**Pull request merged**